cmd/internal/clean/clean.go

```diff
@@ -35,6 +35,9 @@ func init() {
 }
 
 func runCmd(cmd *base.Command, args []string) {
+	defer func() {
```

Comment:
Don't need this recover code. If we don't want to panic, change the original code.